packages/opencensus-core/src/common/validations.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Ensures that an object reference passed as a parameter to the calling
+ * method is not null.
+ *
+ * @param {T} reference An object reference.
+ * @param {string} errorMessage The exception message to use if the check fails.
+ * @returns {T} An object reference.
+ */
+export function checkNotNull<T>(reference: T, errorMessage: string): T {
```

Comment:
If these are validators, can we call them `validateNotNull`, etc?

Reply:
Also, this will throw an error if the `reference` parameter is `0` or `''` (empty string), is that expected? If so this should be documented, but in this case I'd suggest we do a more strict `reference === null || reference === undefined` conditional.

Reply:
Done